{
    "type": "bundle",
    "id": "bundle--abfc6944-f136-4a75-bfb6-7e8114938dd2",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--93b13949-2bb8-46b7-9358-82d13fcef137",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-06-01T00:54:34.283514Z",
            "modified": "2025-06-01T00:54:34.283514Z",
            "name": "CVE-2018-25111",
            "description": "django-helpdesk before 1.0.0 allows Sensitive Data Exposure because of os.umask(0) in models.py.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2018-25111"
                }
            ]
        }
    ]
}